packages/modules/manage-versions/src/components/ManageVersions/VersionsTab/VersionsTab.tsx

```diff
@@ -56,6 +56,19 @@ const VersionsTab = (props: VersionsTabProps) => {
             Header: stringsOverrides.description,
             accessor: "description",
           },
+          {
+            id: "CREATOR",
+            Header: stringsOverrides.user,
+            accessor: "createdBy",
+            maxWidth: 220,
+            Cell: (props: CellProps<NamedVersion>) => {
+              return isLoadingColumn ? (
+                <Text isSkeleton={true}>Fake user cell</Text>
```

Comment:
Change this text.. It still can be read by screen readers. Havfe something like "Loading user info"

Reply:
updated text